src/plugins/intel_cpu/src/exec_network.cpp

```diff
@@ -542,6 +542,7 @@ bool ExecNetwork::CanProcessDynBatch(const InferenceEngine::CNNNetwork &network)
 }
 
 void ExecNetwork::Export(std::ostream& modelStream) {
+    std::const_pointer_cast<ngraph::Function>(_network.getFunction())->set_rt_info(_cfg.exportPropertiesForCaching(), "intel_cpu_config");
```

Comment:
Don't see there "intel_cpu_config" rt_info is used.

Reply:
Removed, thanks